### ISSUE TITLE
tab_univariate doesnt work for non-categorical variables

### ISSUE THREAD

**Issue by epiamsterdam** [R4EPI/sitrep#216]
the function tab_univariate only seems to work if the exposure variables are TRUE/FALSE variables.
```

**Comment by zkamvar** [comment]
This is correct. There are many ways to encode exposure as a binary variable (Yes/No, Oui/Non, 1/0, 1/2, Positive/Negative, etc.), so enforcing these being TRUE/FALSE makes sure that we don't get the exact opposite answer.

This is documented in the R4EPIs website: https://r4epis.netlify.com/training/walk-through/univariate/

```r
## Create vector that specifies the variables we want to convert
binary_vars <- c("case_def", "ptvomit", "ptjaundice", "patient_facility_type")

## Apply str_detect on each of the columns to return TRUE for each element that 
## matches either Confirmed, Oui, or Inpatient
linelist_cc <- linelist_cc %>%
  mutate_at(.vars = binary_vars, 
            .funs = str_detect,
            pattern = "Confirmed|Oui|Inpatient")
```

**Comment by epiamsterdam** [comment]
Agreed. Just that you can then not use tab_univariate to conduct a univariable logistic regression on exposure variables that are not classified as TRUE/FALSE (i.e. continuous).

**Comment by aspina7** [comment]
You wouldn't do it for continuous anyway using this function.
But we should add categorical - and have it automatically take the first or
lowest as reference, then loop for each of the subsets.

Continuous variables would be moving to regression - which should be a
walkthrough on the r4epis page.
